Add Date of addedAt into guide (#7901)
captn3m0/endoflife.date: CONTRIBUTING.md

# Name of the product (mandatory).
title: Timeturner

> # Date of Adding related product into endoflife.date

Reviewer: ```suggestion
# Date when the product was added to endoflife.date (optional).
```
